frontend/src/app/game/local/page.tsx

```diff
@@ -1,369 +1,134 @@
 'use client';
 
-import Join from "../../../components/game/Join"
 import { useSession } from "next-auth/react"
 import { useEffect, useState, useContext } from "react"
 import Link from "next/link"
-import { SocketContext, socket } from "../../../../context/socket"
-import ColorSliderPicker from '../../../components/game/ColorPalette'
+import Customization from "@/components/game/Customization";
 import './styles.css'
 
 export default function GameSettings() {
 
-  const { data: session, status } = useSession();
+	const { data: session, status } = useSession();
 	const [userId, setUserId] = useState(null);
-	const [palette, setPalette] = useState(false);
 
 	const [gameSettings, setGameSettings] = useState({
 		userId: -1,
+
 		background: 0, // 0 - 3 animated, 4 - 5 static
-		
 		palette: 0, // palette: 4 choices
 		bgColor: '#ff0000',
-		
 		opacity: 80,
 		sparks: true,
-		gameDifficulty: 1,
-		pointsToWin: 3,
+
+		gameDifficulty: 4,
+		pointsToWin: 5,
 		powerUps: true
 	});
-	
-	useEffect(() => {
-		localStorage.setItem("gameSettings", JSON.stringify(gameSettings));
-	}, [gameSettings]);
-	
+
+	localStorage.setItem("gameSettings", JSON.stringify(gameSettings));
```

Comment:
I see gameSettings being stored in the localStorage here. If we have switched to the API route, why is this code required here?